documentation/README.md

```diff
@@ -149,10 +149,12 @@ docs.)
 **Assignment**. The `x = 42` expression assigns the value `42` to the variable
 `x`.
 
-**Arithmetic**. The infix operators `+ - * %` work as you'd expect. `%%` tests
-for divisibility, so it returns `true` whenever `%` returns `0`. `divmod` does
-an integer division resulting in a 2-element array `[q, r]` where `q` is the
-quotient and `r` is the reminder.
+**Arithmetic**. The infix operators `+ - * div %` work as you'd expect. (The
```

Comment:
Even though I'm personally fond of the 'temper' part (elegant!) and have no trouble understanding what you mean, my fear is that someone with poor skills in English might, even though it's not probable, get confused. If the div operator would work as expected, it'd include 'non-integer division'. This is of course clarified in the well-tempered sentence afterward, but I think that even though it's highly unlikely, someone could be confused.

However, don't change this! But consider this: a few examples would easily demonstrate the strength, but also the weaknesses in regard to what's possible to do with Mathematics in 007. I also think - very unsure though - this (examples...) falls out of scope for this branch?

Reply:
> Even though I'm personally fond of the 'temper' part (elegant!) and have no trouble understanding what you mean, my fear is that someone with poor skills in English might, even though it's not probable, get confused.

Agreed. I did change the wording. I think you'll like the new phrasing better.

> I also think - very unsure though - this (examples...) falls out of scope for this branch?

Yes, but know that a rewrite is happening. Right now the operators are introduced too briefly. (Also the built-in statement forms, for which there is an issue: #393.) I expect to be making a few passes over the whole documentation before it starts to feel right.